src/no/runsafe/nchat/channel/ChannelManager.java

```diff
@@ -43,17 +44,14 @@ public void registerChatFilter(IChatFilter filter)
 	@Override
 	public IChatChannel getPrivateChannel(ICommandExecutor player1, ICommandExecutor player2)
 	{
-		String player1Name = player1.getName();
-		String player2Name = player2.getName();
-		int cmp = player1Name.compareToIgnoreCase(player2Name);
-		if (cmp == 0)
+		if (player1.equals(player2))
 			return null;
 
 		if (player1 instanceof IPlayer && player2 instanceof IPlayer)
 			if (ignoreHandler.eitherPlayerIsIgnoring((IPlayer) player1, (IPlayer) player2))
 				return null;
 
-		String name = "%" + (cmp < 0 ? player1Name : player2Name) + "-" + (cmp > 0 ? player1Name : player2Name);
+		String name = "%" + (player1.hashCode()) + "-" + (player2.hashCode());
```

Comment:
I actually more had in mind something like `String name = "%" + (player1.hashCode() + player2.hashCode());`
The sum of two player objects hashes should be unique enough for our purposes here.